alphadia/cli.py

```diff
@@ -380,7 +382,7 @@ def run(*args, **kwargs):
             library_path=library_path,
             fasta_path_list=fasta_path_list,
             config=config,
-            custom_quant_dir=custom_quant_dir,
+            quant_path=quant_dir,
```

Comment:
can't we call both of them quant_dir for simplicity?

Reply:
"_path" is more correct and should be used throughout the code base (cf. also the `# TODO rename all output_directory, output_folder => output_path, quant_dir->quant_path (except cli parameter)`)

imho it's fine to name the cli parameter `quant-dir` as this might be more familiar to users .. very early on this should be called `quant_path` then

Reply:
Makes sense to me